[GUIDES] Fix ActiveJob custom serializer typo [ci-skip]

cpjmcquillan opened the pull request:
### Summary
Correct `config/initializer/custom_serializers.rb` to
`config/initializers/custom_serializers.rb`.

### Other information
- Fix typo in guides/source/active_job_basics.md
- Fix 2 typos in guides/source/autoloading_and_reloading_constants.md

p8 commented:
Thanks for the PR @cpjmcquillan !
There are two more in guides/source/autoloading_and_reloading_constants.md
Could you fix those as well?

Commit:
[GUIDES] Fix ActiveJob custom serializer typo

Correct `config/initializer/custom_serializers.rb` to
`config/initializers/custom_serializers.rb`.

- Fix typo in guides/source/active_job_basics.md
- Fix 2 typos in guides/source/autoloading_and_reloading_constants.md

cpjmcquillan commented:
> Thanks for the PR @cpjmcquillan !
> There are two more in guides/source/autoloading_and_reloading_constants.md
> Could you fix those as well?
> 
> ```
> guides/source/autoloading_and_reloading_constants.md
> 131:# config/initializer/custom_serializers.rb
> 154:# config/initializer/custom_serializers.rb
> ```

No problem. 

Done and I have tidied up the commits 👍.

p8 commented:
Thanks @cpjmcquillan ! 🙌
